src/App.js

```diff
@@ -1,16 +1,133 @@
 import React, { Component } from 'react'
-import logo from './logo.svg'
 import './App.css'
+import PlayMenu from './PlayMenu'
+import Naming from './Naming'
 
 class App extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      game: {
+        gameState: 'naming',
```

Comment:
Would prefer to see important magical strings defined as a `const` at the top of the file. That way your linter can catch errors. For example, if you did `gameState: 'namnig'`, you're not going to catch that until runtime. However if you did

```js
const NAMING = 'naming'
...
  game: {
    gameState: NAMNIG,
```
Then your linter is going to go "Hey, wait a minute, `NAMNIG` is not defined.